.githooks/scripts/pylint.sample

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# Ensure the modified files conform to the style enforced by the "Style" CI
+# test. You may change the `pylint_cmd` variable to set the path of your pylint
+# executable.
+
+pylint_cmd=pylint
+
+set -euo pipefail
+cd -e -P -- "$(dirname -- "$0")/../../../"
+base_branch=$(git branch --show-current | sed -E -e 's/(master|(saas-)?\d\d.\d)/\1/')
```

Comment:
`git branch --show-current` is only present since [git 2.22 ](https://github.com/git/git/commit/0ecb1fc7269e15be890870937b8b639c987abd08). Debian current stable only provides [2.20](https://packages.debian.org/buster/git) and Ubuntu Focal gives 20.04.
So `git rev-parse --abbrev-ref HEAD` should be more portable as stated [here](https://stackoverflow.com/a/1418022)

Reply:
Also not sure about the regex:
If you want to output only the matched pattern, I think the egrep is command is simpler:
`egrep -o '(master|(saas-)?\d+\.\d)'`

The `. ` is escaped in order to be matched.